test/sharness/t0180-p2p.sh

```diff
@@ -73,10 +73,18 @@ test_server_to_client() {
 
 spawn_sending_server
 
-test_expect_success 'S->C Setup client side' '
+test_expect_success 'S->C(/ipfs/peerID) Setup client side' '
   ipfsi 1 p2p forward /x/p2p-test /ip4/127.0.0.1/tcp/10102 /ipfs/${PEERID_0} 2>&1 > dialer-stdouterr.log
 '
 
+test_expect_success 'S->C Setup(dnsaddr/addr) client side' '
+  ipfsi 1 p2p forward /x/p2p-test /ip4/127.0.0.1/tcp/10103 /dnsaddr/bootstrap.libp2p.io/ 2>&1 > dialer-stdouterr.log
+'
```

Comment:
It would be great if we found a nicer way to test this

Reply:
I just imitated the test `ipfsi 1 p2p forward /x/p2p-test /ip4/127.0.0.1/tcp/10102 /ipfs/${PEERID_0} 2>&1 > dialer-stdouterr.log` . And use `test_expect_success`  to judge the test success.

Reply:
And i had add `test_must_be_empty dialer-stdouterr.log` to judge it succeed

Reply:
You can also call `ipfsi 1 p2p ls` and check if it's output contains resolved listener (`grep '/x/p2p-test /ip4/127.0.0.1/tcp/10104 /'` or something like that)

Reply:
@magik6k Thx for advice , i have update the test. Please help me review it again,